test/form_builder/general_test.rb

```diff
@@ -217,6 +217,11 @@ def with_custom_form_for(object, *args, &block)
     assert_select 'form select#user_age.select'
   end
 
+  test 'builder does not generate url fields for columns that contain only the letters url' do
+    with_form_for @user, :hourly
+    assert_no_select 'form input#user_url.string.url'
```

Comment:
Wdyt to add a assert to match the generated type?

Reply:
Yeah, let's check the generated type.

Reply:
@sringling Are you willing include this assert? after that we can merge it 😅

Reply:
Not 100% sure I did what you meant, but I see the other tests tend to have these same 3 so I think I got it.

Reply:
Now, it's ok for me 👍